include/libu64/gfxprint.h

```diff
@@ -3,33 +3,105 @@
 
 #include "ultra64.h"
 #include "color.h"
+#include "libc64/aprintf.h"
 
-typedef struct gfxprint {
-    /* 0x00 */ char unk00[0x30];
-} gfxprint; // size = 0x30
-
-// void gfxprint_setup();
-void gfxprint_color(gfxprint* this, u8 r, u8 g, u8 b, u8 a);
-// void gfxprint_locate();
-void gfxprint_locate8x8(gfxprint* this, s32 arg1, s32 arg2);
-// void gfxprint_setoffset();
-// void gfxprint_putc1();
-// void gfxprint_putc();
-// void gfxprint_write();
-// void gfxprint_puts();
-// void gfxprint_prout();
+typedef struct gfxprint_obj  {
```

Comment:
Could you use the same name for the struct name at the top and the bottom?
```suggestion
typedef struct gfxprint  {
```